pkg/credentials/certchain.go

```diff
@@ -4,13 +4,13 @@ import (
 	"os"
 )
 
-const (
+var (
 	// RootCertFilename is the filename that holds the root certificate.
-	RootCertFilename = "ca.crt"
+	RootCertFilename string = "ca.crt"
```

Comment:
Where is the default path for certs?

Reply:
each binary is defining it, a bit of duplication there,
https://github.com/dapr/dapr/blob/master/cmd/sentry/main.go#L42
https://github.com/dapr/dapr/blob/master/cmd/placement/config.go#L27
https://github.com/dapr/dapr/blob/master/cmd/operator/main.go#L39

injector is getting them from the Secret directly, no need to mount it as volume,
https://github.com/dapr/dapr/blob/master/pkg/injector/pod_patch.go#L320

Reply:
I would like to put path in `pkg/credentials` so that we only one config entry.

Reply:
totally agree, I just changed as few code as possible just because I don't know much about dapr, this proposal, for me, is just to get rid of this workaround: https://github.com/safanaj/dapr-cert-transformer/blob/master/main.go#L63

I am running this workaround because I used to manage all the tls certs with cert-manager, as also wished in here #3968